stockflux-watchlist/public/app.dev.json

```diff
@@ -1,37 +1,37 @@
 {
-    "runtime": {
-        "arguments": "--enable-aggressive-domstorage-flushing",
-        "version": "9.61.38.40"
-    },
-    "startup_app": {
-        "name": "stockflux-watchlist",
-        "uuid": "33aa9062-9eb0-4875-b819-c90f38ef03eb",
-        "url": "http://localhost:3000/index.html",
-        "applicationIcon": "http://localhost:3000/favicon.ico",
-        "autoShow": true,
-        "defaultWidth": 230,
-        "defaultHeight": 600,
-        "maxWidth": 230,
-        "minWidth": 230,
-        "minHeight": 300,
-        "frame": false,
-        "contextMenu": true
-    },
-    "services": [
-        {
-            "name": "layouts",
-            "config": {
-                "features": {
-                    "tab": false
-                }
-            }
+  "runtime": {
+    "arguments": "--enable-aggressive-domstorage-flushing",
+    "version": "9.61.38.40"
```

Comment:
Feel free to bump this up to latest stable (e.g. 10.66.39.43)

Reply:
Fixed -> a03e818b1a82eea4f6e3cba19b30cd030588ee23